pkg/virt-handler/node-labeller/config.go

```diff
@@ -58,11 +65,16 @@ func (n *NodeLabeller) loadConfig() (Config, error) {
 		return config, nil
 	}
 
-	if value, ok := cm.Data["cpu-plugin-configmap.yaml"]; ok {
-		err := yaml.Unmarshal([]byte(value), &config)
-		if err != nil {
-			return config, err
-		}
+	if _, ok := cm.Data[virtconfig.ObsoleteCPUsKey]; ok {
+		config.ObsoleteCPUs = n.clusterConfig.GetObsoleteCPUs()
+	} else {
+		config.ObsoleteCPUs = defaultObsoleteCPUs
+	}
+
+	if _, ok := cm.Data[virtconfig.MinCPUKey]; ok {
+		config.MinCPU = n.clusterConfig.GetMinCPU()
```

Comment:
would it make sense to move the default obsoleteCPUs and minCPU values to the cluster config logic.  that way you'd just call `n.clusterConfig.GetMinCPU()` and get the most recent value regardless if it is the default or not.